lib/ansible/plugins/shell/__init__.py

```diff
@@ -173,6 +173,8 @@ def append_command(self, cmd, cmd_to_append):
 
         return cmd
 
-    def wrap_for_exec(self, cmd):
+    def wrap_for_exec(self, cmd, env_string):
```

Comment:
`wrap_for_exec` used only in lib/ansible/plugins/action/script.py, so I think that changing the signature shouldn't broke a lot of things.